tests/lib/external/snapd-testing-tools/utils/log-parser

```diff
@@ -82,9 +48,10 @@ class Result:
     The results can be: Successful, failed and aborted
     """
 
-    def __init__(self, result_type, level, stage, number, date, time,
-                 detail, source_line):
-        self.type = RESULT
+    def __init__(
```

Comment:
@zyga wrote:
There's a lot of string arguments that are somewhat confusing. What is level? What is number?

Please document those.